sale_rental/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2014-2021 Akretion France (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2016-2021 Sodexis (http://sodexis.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Rental",
+    "version": "17.0.1.0.0",
+    "category": "Sales",
+    "license": "AGPL-3",
+    "summary": "Manage Rental of Products",
+    "author": "Akretion, Sodexis, Odoo Community Association (OCA)",
+    "maintainers": ["alexis-via"],
+    "website": "https://github.com/OCA/vertical-rental",
+    "depends": ["sale_start_end_dates", "sale_stock", "sales_team"],
```

Comment:
sales_team is not needed here, since both other modules depends on sale which in turn depends on sales_team.
On the other side, I think sale_management should be added here as a dependency, since if not installed you will not get the Sales menu shown so you cannot access any of the functionality added by this module.
```suggestion
    "depends": ["sale_start_end_dates", "sale_stock", "sale_management"],
```